Zapper-fi/studio
# feat(mux): Add lev-trades ContractPositionFetcher and add BalanceFetcher

## Description

 Add lev-trades ContractPositionFetcher and add BalanceFetcher

## Checklist


## How to test?
1. edit/create `.env` adding `ENABLED_APPS=mux`
2. run `pnpm dev` to spin up nest server
3. visit dev endpoints below to see result to integrate mux
* http://localhost:5001/apps/mux/positions?groupIds[]=lev-trades&network=arbitrum
* http://localhost:5001/apps/mux/positions?groupIds[]=lev-trades&network=avalanche
* http://localhost:5001/apps/mux/positions?groupIds[]=lev-trades&network=binance-smart-chain
* http://localhost:5001/apps/mux/positions?groupIds[]=lev-trades&network=fantom
* http://localhost:5001/apps/mux/balances?addresses[]=0x8371f1525fB09C2915828c6c2718f30095A6D094&network=arbitrum
* http://localhost:5001/apps/mux/balances?addresses[]=0x8371f1525fB09C2915828c6c2718f30095A6D094&network=avalanche
* http://localhost:5001/apps/mux/balances?addresses[]=0x8371f1525fB09C2915828c6c2718f30095A6D094&network=binance-smart-chain
* http://localhost:5001/apps/mux/balances?addresses[]=0x8371f1525fB09C2915828c6c2718f30095A6D094&network=fantom
